transact/alchemy_sendGasOptimizedTransaction.yaml

```diff
@@ -37,6 +37,9 @@ paths:
                   $ref: ../components/schemas.yaml#/Id
                 jsonrpc:
                   $ref: ../components/schemas.yaml#/JsonRpc
+                method:
+                  $ref: ../components/schemas.yaml#/Method
+                  default: alchemy_sendGasOptimizedTransaction
```

Comment:
@vahnag do we need this?

Reply:
yes, this is required for all JSONRPC endpoints